cg/models/cg_config.py

```diff
@@ -169,6 +169,10 @@ class RnafusionConfig(CommonAppConfig):
     tower_pipeline: str
 
 
+class TaxprofilerConfig(CommonAppConfig):
+    pass
```

Comment:
I would either finish this in the PR or move it completely to the next PR

Reply:
I will remove this class in this PR then.